homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html

```diff
@@ -0,0 +1,130 @@
+            when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every
+            pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business
+            it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man
+            therefore always holds in these matters to this principle of selection: he rejects pleasures to secure
+            other greater pleasures, or else he endures pains to avoid worse pains.
+
+            On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and
+            demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the
+            pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through
+            weakness of will, which is the same as saying through shrinking from toil and pain. These cases are
+            perfectly simple and easy to distinguish. In a free hour, when our power of choice is untrammeled and
+            when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every
+            pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business
+            it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man
+            therefore always holds in these matters to this principle of selection: he rejects pleasures to secure
+            other greater pleasures, or else he endures pains to avoid worse pains.
+
+            On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and
+            demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the
+            pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through
+            weakness of will, which is the same as saying through shrinking from toil and pain. These cases are
+            perfectly simple and easy to distinguish. In a free hour, when our power of choice is untrammeled and
+            when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every
+            pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business
+            it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man
+            therefore always holds in these matters to this principle of selection: he rejects pleasures to secure
+            other greater pleasures, or else he endures pains to avoid worse pains.
+        </p>
+    </div>
+  </div>
+  <script defer src="app.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/6.6.3/rxjs.umd.min.js"
```

Comment:
You need to import third-party scripts before your own scripts. So please, change the order

Everything else is great, I can merge request after this little tweak

Reply:
Done